stripe/src/main/java/com/stripe/android/payments/StripeBrowserLauncherActivity.kt

```diff
@@ -30,10 +27,6 @@ internal class StripeBrowserLauncherActivity : AppCompatActivity() {
         StripeBrowserLauncherViewModel.Factory(application)
```

Comment:
Can you send the args to the factory?  Then rather than calling launch(viewModel.convertArgToIntent) you can just retrieve it?

It feels slightly weird for the viewModel to be making intents, I wonder if it should be just a factory, utility, or extension that just does the conversion?   I don't have strong opinions though

Reply:
I could inject args as well, but it would be a little awkward so I went with this approach.

Why do you think a ViewModel shouldn't create an Intent?

Reply:
Intent just feels very much like an Activity sort of thing 🤷 .  It doesn't really need or use anything in the viewModel class (none of the other functions or member variables of that class.

It's ok as is, just a thought.

Reply:
I think it's a coin-flip logically, but easier to test if in the ViewModel.

I think if the Intent depends on a `Context` it might need to live in the Activity.